tests/performance/realtime.go

```diff
@@ -80,8 +80,11 @@ var _ = SIGDescribe("CPU latency tests for measuring realtime VMs performance",
 	})
 
 	It("running cyclictest and collecting results directly from VM", func() {
-		vmi := tests.NewRandomVMIWithEphemeralDiskAndUserdata(cd.ContainerDiskFor(cd.ContainerDiskFedoraRealtime), tuneAdminRealtimeCloudInitData)
-		byConfiguringTheVMIForRealtime(vmi, "")
+		vmi := libvmi.NewFedora(
```

Comment:
I don't think we should use `NewFedora` and I am not sure this work as you expect. Doesn't the two `libvmi.WithContainerImage` conflict?